### PR TITLE
Mpuncel/rc schedule test

### PR DESCRIPTION
**Pull request by mpuncel**

### CODE REVIEW

**Comment by mpuncel** [comment]
i was paranoid and wrote this, figured i may as well check it in

File: pkg/rc/replication_controller_test.go [hunk 1]
```diff
@@ -185,7 +186,7 @@ func TestSchedule(t *testing.T) {
 	rc.WatchDesires(quit)
 
 	rcStore.SetDesiredReplicas(rc.ID(), 1)
-	numNodes := waitForNodes(t, rc, 1)
+	numNodes := waitForNodes(t, rc, 7)
```
Comment (petertseng) [inline pkg/rc/replication_controller_test.go]: should this really have changed? waits for 7 (I assume that times out), then asserts 1

File: pkg/rc/replication_controller_test.go [hunk 2]
```diff
@@ -528,3 +529,70 @@ func TestReservedLabels(t *testing.T) {
...
+	go func() {
+		defer wg.Done()
+		for err := range errors {
+			t.Error(err)
+		}
+	}()
+
+	rcStore.SetDesiredReplicas(rc.ID(), 7)
+
+	numNodes := waitForNodes(t, rc, 1)
```
Comment (petertseng) [inline pkg/rc/replication_controller_test.go]: waits for 1 (I assume this times out) then asserts 7?

File: pkg/rc/replication_controller_test.go [hunk 3]
```diff
@@ -528,3 +529,70 @@ func TestReservedLabels(t *testing.T) {
...
```
Comment (petertseng) [inline pkg/rc/replication_controller_test.go]: waits for 1 (I assume this times out) then asserts 7?

File: pkg/rc/replication_controller_test.go (hunk 1 above)
Comment (mpuncel) [inline pkg/rc/replication_controller_test.go]: i have no idea how this happened

**Comment by mpuncel** [comment]
fixed all the weirdness in the test

**Comment by petertseng** [comment]
Actually

**Comment by petertseng** [comment]
You should move the 1->7 change and 7->1 change into the other commit

**Comment by petertseng** [comment]
Oh, although you're going to have to rebuild again anyway, you're getting vetted for using `%s` on `i`.

**Pull request merged**